pkgs/tools/graphics/mangohud/default.nix

```diff
@@ -140,6 +162,10 @@ in stdenv.mkDerivation rec {
     wrapProgram "$out/bin/mangohud" \
       --prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ addOpenGLRunpath.driverLink ]} \
       --prefix XDG_DATA_DIRS : "$out/share"
+  '' + lib.optionalString (gamescopeSupport && stdenv.hostPlatform.system == "x86_64-linux") ''
```

Comment:
Was the `stdenv.hostPlatform.system == "x86_64-linux"` part copied from the `postInstall` section? I think we'd want to wrap this regardless of platform like we do for the mangohud binary.

Reply:
The `mangoapp` executable is only built on 64-bit platforms, and it doesn't exist otherwise. Let's check for its existence in the script instead so it's less likely to break.